### DISCUSSION TITLE
Monolithic archive like in soundfonts

### DISCUSSION THREAD

**Discussion by redtide** [sfztools/sfizz#1214] [Requests]
Add the possibility to read all the files from a single uncompressed archive for streaming.
It requires some choice whether to use a single file including sfz file(s) or (also) be able to edit an external sfz file keeping samples in the archive, similar to a cue + bin cd image.

**Comment by kinwie** [comment]
This will be very useful for large library which samples count more than 10.000 samples that can be stored in a single zip or iso, and possibly can also increase sample accessing time
